### PR TITLE
Chore: Update base image to Debian bookworm

### PR DESCRIPTION
**Pull request by stumpylog**
## Proposed change

Debian Bookworm is now released and a Python base image using it is availbile.  This PR upgrades us to this image, while retaining Python 3.9.  This brings in notable upgrades to many of the packages utilized by ourselves and other libraries.

| Package | Old | New | Changelog |
| --------- | ---- | ----- | ------------ |
|  tesseract |  4.1.1 | 5.3.0 | [4.1.1](https://tesseract-ocr.github.io/tessdoc/ReleaseNotes.html#v411) .. [5.3.0](https://tesseract-ocr.github.io/tessdoc/ReleaseNotes.html#v530) |
| ghostscript | 9.53.3 | 10.0.0 | [news](https://ghostscript.readthedocs.io/en/latest/News.html) |
| unpaper | 6.1 | 7.0.0 | [news](https://github.com/unpaper/unpaper/blob/main/NEWS) |
| poppler-utils | 20.09.0 | 22.12.0 | [news](https://poppler.freedesktop.org/releases.html) |
| mariadb-client | 10.5 | 10.11 | [changelog](https://mariadb.com/kb/en/release-notes/) |
| libpq5 | 13.10 | 15.3 | [changelog](https://www.postgresql.org/docs/release/) |

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] Other (please explain) - Upgrade image base

## Checklist:

- [ ] I have read & agree with the [contributing guidelines](https://github.com/paperless-ngx/paperless-ngx/blob/main/CONTRIBUTING.md).
- [ ] If applicable, I have tested my code for new features & regressions on both mobile & desktop devices, using the latest version of major browsers.
- [ ] If applicable, I have checked that all tests pass, see [documentation](https://docs.paperless-ngx.com/development/#back-end-development).
- [ ] I have run all `pre-commit` hooks, see [documentation](https://docs.paperless-ngx.com/development/#code-formatting-with-pre-commit-hooks).
- [ ] I have made corresponding changes to the documentation as needed.
- [ ] I have checked my modifications for any breaking changes.

### CODE REVIEW

**Comment by stumpylog** [comment]
That's why it's a draft...

**Comment by suhrmann** [comment]
I overlooked that. 🙄 I was too hyped testing it...

**Comment by stumpylog** [comment]
I know the feeling.  I'm watching the PR which should add the bookworm image to the Docker image library, then this should be ready to go in

File: Dockerfile [hunk 1]
```diff
@@ -5,7 +5,7 @@
 # Purpose: Compiles the frontend
 # Notes:
 #  - Does NPM stuff with Typescript and such
-FROM --platform=$BUILDPLATFORM node:16-bullseye-slim AS compile-frontend
+FROM --platform=$BUILDPLATFORM docker.io/node:16-bullseye-slim AS compile-frontend
```
Comment (suhrmann) [inline Dockerfile:8]: Use [`node:16-bookworm-slim`](https://hub.docker.com/layers/library/node/16-bookworm-slim/images/sha256-1362db52e34df6d0fd67619257c4169c0b39c6c0e8d3ec3c4b3c843bf7e3e2f0?context=explore) instead to keep it consistent to `python:3.9-slim-bookworm`?
But in fact that should not make much difference...

Comment (stumpylog) [reply Dockerfile:8]: Good point.  I wouldn't expect any difference, but consistent is best.

**Comment by hendrik1120** [comment]
bookworm image has just been released

**Comment by stumpylog** [comment]
Almost.  The ARMv7 images are [still building](https://doi-janky.infosiftr.net/job/multiarch/view/images/view/python/job/arm32v7/job/python/).

**Comment by stumpylog** [comment]
The base images and our own have built.  Any testing on this would be great.

**Comment by shamoon** [comment]
Exciting!

I have to run out but I just spun it up to see and getting `PermissionError: [Errno 13] Permission denied: 'jbig2'` on consumption. This is my typical [docker] dev setup (pretty standard, local volume mounts etc) and reverting back to `dev` seems to work fine again. I can try to dig into it later but figured I'd mention.

**Comment by shamoon** [comment]
Thanks for the lightning fast fix! Nice, yea that did the trick.

So far all seems copasetic, .pdfs, .doc, emails, logs look normal. Anything else you'd want specifically to check?

**Comment by stumpylog** [comment]
Nothing particular I can think of to test.  I used Postgres and MariaDB and didn't notice any problems connecting, so the underlying library upgrades seem alright.  it maybe seems a little quicker/smoother when consuming?

Overall, seems to be working to me.

**Review by shamoon — APPROVED** [review]
Awesome, with the other deps updates as well I'm really hoping this brings some benefits / stability

**Pull request merged**